test_utilities/bin/flutter_test_runner.sh

```diff
@@ -23,7 +23,7 @@ pushd $1 > /dev/null
 
 flutter packages get
 flutter analyze
-flutter format --line-length=120 --set-exit-if-changed lib/ test/
+dart format --line-length=120 --set-exit-if-changed lib/ test/
```

Comment:
Not sure if we should follow dart or flutter. Here is wiki from flutter formatting, seems `flutter` is more preferred?

Reply:
I'll land these change as a separate PR. There is a deprecation notice, seems like flutter functionality will be removed.